src/gui/accountsettings.cpp

```diff
@@ -403,6 +426,16 @@ void AccountSettings::slotOpenCurrentFolder()
     }
 }
 
+void AccountSettings::slotOpenCurrentLocalSubFolder()
+{
+    QModelIndex selected = ui->_folderList->selectionModel()->currentIndex();
```

Comment:
I think that's fine.

Altough there is no need to go trough the selectionModel:  `ui->_folderList->currentIndex()` should work